template_scripts/generate_schemas.sh

```diff
@@ -1,12 +0,0 @@
-#!/usr/bin/env bash
-
-#GITHUB_TOKEN=<INSERT> \
-#ZENHUB_TOKEN=<INSERT> \
-poetry run python3 -m sgqlc.introspection \
-  --exclude-deprecated \
-  --exclude-description \
-  -H "Authorization: bearer $GITHUB_TOKEN" \
-  https://api.github.com/graphql \
-  github_schema.json
```

Comment:
@nathan-at-least I think a replacement for this script is needed; if you haven't generated github_schema.json none of the rest of this will work.